examples/sdk-usage/build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 }
 
 group 'io.opentelemetry'
-version '0.3.0-SNAPSHOT'
+version '0.3.0'
```

Comment:
Why changing this?

Reply:
Since we have a release, I am also in favor of using `0.3.0`.
When we will introduce/change SDK APIs, we should change this to `0.4.0-SNAPSHOT`

Reply:
I think we never want to push this to maven. Having this without snapshot may cause mistakes and push them to maven central

Reply:
I was thinking to have this as 0.4.0-Snapshot same as API/sdk

Reply:
I did mostly to keep the invariant of other examples. I'm fine using `SNAPSHOT` for now.

I think it's a minor thing, so lets go with `0.4.0-SNAPSHOT` for now - if we face any problem, we will realize it soon and will fix that by then ;)

Reply:
Oh, actually just realized examples depend on published artifacts (see log below). So we can have `0.4.0-SNAPSHOT` as version **but** consume whatever published artifacts we have (`0.3.0`). Sounds good enough for you @bogdandrutu ?

```
* What went wrong:
Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not find io.opentelemetry:opentelemetry-api:0.3.0-SNAPSHOT.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/io/opentelemetry/opentelemetry-api/0.3.0-SNAPSHOT/maven-metadata.xml
       - https://repo.maven.apache.org/maven2/io/opentelemetry/opentelemetry-api/0.3.0-SNAPSHOT/opentelemetry-api-0.3.0-SNAPSHOT.pom
       - https://repo.maven.apache.org/maven2/io/opentelemetry/opentelemetry-api/0.3.0-SNAPSHOT/opentelemetry-api-0.3.0-SNAPSHOT.jar
```

Reply:
Ok, so let's clarify:
1. `version '0.3.0'` causes the generated Jar to have the version `0.3.0` which is something that I try to avoid because in case of a mistake and these get pushed to maven will not end up in the maven central repo.
1.  You are using `version` for the opentelemetry dependencies, which is not correct because `version` is a special keyword and represents the version of this target. So you need to define a new variable `opencensusVersion = '0.3.0'`, and there we use the released  `'0.3.0'` version.

Reply:
Also the version for the example jar can stay at 0.1.0-SNAPSHOT always and don't have to change that because we don't intend to ever release them to maven or do a release at all.

Reply:
1) and 2) correct.

So lets keep them separated **and** lets keep the versions are as `0.1.1-SNAPSHOT` to signal they remain independent (and non published).

Reply:
I would go with `0.1.0-SNAPSHOT` not `0.1.1-SNAPSHOT`

Reply:
(I had meant 0.1.0 😓 )

Reply:
Done.